benchmarks/benchmarks/stats_sampling.py

```diff
@@ -130,24 +145,60 @@ def setup(self, dist, c):
         with np.testing.suppress_warnings() as sup:
             sup.filter(RuntimeWarning)
             try:
-                self.rng = stats.TransformedDensityRejection(
+                self.rng = sampling.TransformedDensityRejection(
                     dist, c=c, random_state=self.urng
                 )
-            except stats.UNURANError:
+            except sampling.UNURANError:
                 # contdist3 is not T-concave for c=0. So, skip such test-cases
                 raise NotImplementedError(f"{dist} not T-concave for c={c}")
 
     def time_tdr_setup(self, dist, c):
         with np.testing.suppress_warnings() as sup:
             sup.filter(RuntimeWarning)
-            stats.TransformedDensityRejection(
+            sampling.TransformedDensityRejection(
                 dist, c=c, random_state=self.urng
             )
 
     def time_tdr_rvs(self, dist, c):
         self.rng.rvs(100000)
 
 
+class SimpleRatioUniforms(Benchmark):
+
+    param_names = ['dist', 'cdf_at_mode']
+
+    params = [allcontdists, [0, 1]]
+
+    def setup(self, dist, cdf_at_mode):
+        self.urng = np.random.default_rng(0xfaad7df1c89e050200dbe258636b3265)
+        try:
+            if cdf_at_mode:
+                cdf_at_mode = dist.cdf(dist.mode)
+            else:
+                cdf_at_mode = None
+            self.rng = sampling.SimpleRatioUniforms(
+                dist, mode=dist.mode,
+                cdf_at_mode=cdf_at_mode,
+                random_state=self.urng
+            )
+        except sampling.UNURANError:
+            raise NotImplementedError("{dist} not T-concave")
```

Comment:
does Python have implicit f-strings for error messages these days or is this missing the `f` prefix? Or is this meant to be a literal use of braces?

Reply:
Oops, this is supposed to be an f-string. I will fix this in a follow-up PR. Thanks for noticing!